src/frontend/web_application/src/components/AuthorAvatar/presenter.jsx

```diff
@@ -9,18 +11,24 @@ class AuthorAvatar extends PureComponent {
   static propTypes = {
     message: PropTypes.shape({}).isRequired,
     isSelected: PropTypes.bool,
+    size: PropTypes.oneOf([SIZE_SMALL, SIZE_MEDIUM]),
   };
   static defaultProps = {
     isSelected: false,
+    size: SIZE_SMALL,
   };
 
   render() {
-    const { isSelected } = this.props;
+    const { isSelected, size } = this.props;
     const participant = getAuthor(this.props.message);
 
     return (
-      <div className="m-author-avatar">
-        <ParticipantIconLetter className="m-author-avatar__letter" isSelected={isSelected} participant={participant} />
+      <div className={classnames('m-author-avatar', { 'm-author-avatar--small': size === SIZE_SMALL, 'm-author-avatar--medium': size === SIZE_MEDIUM })}>
+        <ParticipantIconLetter
+          className={classnames('m-author-avatar__letter', { 'm-author-avatar-small__letter-': size === 'small', 'm-author-avatar--medium__letter': size === 'medium' })}
```

Comment:
typo here: `m-author-avatar-small__letter-` should be `m-author-avatar--small__letter`